app/utils/prepareAnswer.js

```diff
@@ -90,9 +90,20 @@ function prepareDailyStatsAnswer(body, n) {
     return data
 }
 
+function prepareTestingResourceAnswer(stateResources) {
+    var data = `\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, showing ${stateResources.length} results\n`
+
+    stateResources.forEach(stateResource => {
+        data += `\n<b>${stateResource.nameoftheorganisation}, ${stateResource.city}\n</b>Description: ${stateResource.descriptionandorserviceprovided}.\n<i>Phone Number: ${stateResource.phonenumber}</i>\n<i>Website: ${stateResource.contact}</i>\n`
```

Comment:
I want website but not the website review. You can disable that using https://core.telegram.org/bots/api#sendmessage